BibBuddy/app/src/main/java/de/bibbuddy/BookNotesView.java

```diff
@@ -35,17 +40,43 @@
   private View view;
   private Context context;
   private BookNotesViewModel bookNotesViewModel;
-  private RecyclerView recyclerView;
   private NoteRecyclerViewAdapter adapter;
   private Long bookId;
-  private List<NoteItem> noteList;
 
+  private ActivityResultLauncher<String> requestPermissionLauncher;
+
+  private List<NoteItem> noteList;
   private BookDao bookDao;
   private NoteDao noteDao;
-
   private ExportBibTex exportBibTex;
   private SortCriteria sortCriteria;
 
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setupPermissionLauncher();
+  }
+
+  /* Register permissions callback, which handles the user's response to the
+   system permissions dialog. Save the return value, an instance of
+   ActivityResultLauncher, as an instance variable.
+   */
```

Comment:
Proper formatting for the comment :)
```suggestion
  /** Register permissions callback, which handles the user's response to the
    * system permissions dialog. Save the return value, an instance of
    * ActivityResultLauncher, as an instance variable.
    */
```